draft-ietf-tls-external-psk-importer.md

```diff
@@ -99,8 +98,10 @@ identities if not imported correctly. Endpoints may incrementally deploy PSK imp
 by offering non-imported keys for TLS versions prior to TLS 1.3. Non-imported and imported PSKs
 are distinct since their identities are different on the wire. See {{rollout}} for more details.
 
-Clients which import external keys TLS MUST NOT use these keys for any other purpose.
-Moreover, each external PSK MUST be associated with at most one hash function.
+Clients which import external keys for TLS MUST NOT use these keys for any other
+purpose. Moreover, each external PSK MUST be associated with at most one hash function,
```

Comment:
Does this mean you must not use the key you are feeding into the importer or the key you derive from the importer?

Reply:
The former, though that may be a bit too strong of a requirement.

Reply:
As I understand the technical requirement it's that you must only use it with HKDF with that hash and a unique label. So if, for instance, you were to HKDF with a label that guaranteed non-collision with importers, that would be OK, right

Reply:
That's right, and that's what the security considerations discuss. (Are you looking for a change in text here?)